test.js

```diff
@@ -0,0 +1,2480 @@
+// for(let i = strNum.length-1; i >= 0; i--){
+//     strNumRevers.push(strNum[i])
+// }
+// console.log(strNumRevers.join(''))
+
+//////58
+// let r = 45
+// r += 3
+// r -= 8
+// console.log(r)
+
+////60.5
+// let num1 = 3;
+// let num2 = ++num1;///let num 2 = (num1 = num1 + 1)
+// alert(num1);///4
+// alert(num2);///4
+
+// let num1 = 3;
+// let num2 = (num1 = num1 + 1)
+// alert(num1);///4
+// alert(num2);///4
+
+// let num1 = 3;
+// let num2 = num1++;
+// alert(num2);
+// alert(num1);
+
+// let num1 = 3;
+// let num2 = num1
+// num1 = num1 + 1 
+// alert(num2);
+// alert(num1);
+
+////61
+// let a = Math.PI
+// console.log(+a.toFixed(a))
+ 
+///62
+// alert("вам"+' '+prompt("сколько вам лет?")+' '+"лет")
+///63
+// let q1 = prompt('number 1')
+// let q2 = prompt('number 2')
+// alert(+q1 + +q2)
+
+// let q1 = prompt('a')
+// let q2 = prompt('b')
+// s= q1 * q2
+// alert(`s=${s}`)
+
+
+// let q1 = prompt('a')
+// let q2 = prompt('b')
+// p= q1*2 + q2*2////преобразованіе в чісло проісходіт за счет *
+
+// alert(`p=${p}`)
+
+//////64
+// let w = 'hi'
+// document.write('<i>'+w+'</i>')
+// document.write('1<br>')
+// document.write('2<br>')
+// document.write('3<br>')
+// document.write('4<br>')
+// document.write('5<br>')
+///65
+// let num1 = 1;
+// let num2 = 2;
+// let sum = num1+num2
+// console.log('сумма: ' + sum);
+
+// let a = 1;
+// let b = 2;
+// let c
+// console.log(c);
+// console.log(a + b + c);
+
+// let num = '123';
+// let sum = +num[0] + +num[1] + +num[2];
+// console.log(sum);
+
+// let num = 123;
+// let strNum = String(num)
+// console.log(+(strNum[0]));
+
+// let a = 0;
+// console.log(++a);
+
+// let num = String(123);
+// console.log(num.length);
+
+// let a = 24 * 60 * 60;
+// console.log(a);
+
+// let num = 123;
+// let str = String(num);
+// console.log(str.length);
+
+// let num = 123;
+// let str = String(num);
+// console.log(str[str.length-1]);
+
+// let num = 123;
+// let str = String(num);
+// console.log(str.length);
+
+// let num = 123;
+// let str = String(num);
+// console.log(str[str.length-1]);
+
+// let a = '123';
+// let b = '456';
+// let s = Number(a) + +b;
+// console.log(s);
+
+// let aaa = 1;
+// let bbb = 2;
+// let ccc = 3;
+// console.log(aaa + bbb + ccc);
+
+///66
+// alert(60*60*24*30)
+// alert(60*60*24*365)
+// alert(60*24)
+// alert(60*24*365)
+
+// alert(1*1024*1024*1024*10)
+// alert(1*1024*1024*1024*1024)
+// alert(1*1024*1024*1024)
+
+///67
+// let r = 3.14
+// let s = Math.PI*r**2
+// alert(s)
+// alert(3.14**3)
+
+// let a = 3.14
+// let s = a**2
+// alert(s)
+// alert(3.14**2)
+
+// let a = 5
+// let b = 3
+// let s = a*b
+// alert(s)
+
+// let a = 5
+// let b = 3
+// let p = (a+b)*2
+// alert(p)
+
+// let c = 0
+// let f = c * 9/5 + 32;
+// alert(f)
+
+// let f = 32
+// let c = (f-32) * 5/9;
+// alert(c)
+
+//////69
+// let s = [1, 2, 3]
+// console.log(s[0]+s[1]+s[2])
+
+// let w = ['a', 'b', 'c', 'd']
+// console.log(`'${w[0]+'+'+w[1]+'+'+w[2]+'+'+w[3]}'`)
```

Comment:
вось так гэта павінна выглядаць

```suggestion
// let w = ['a', 'b', 'c', 'd']
// console.log(`${w[0]}+${w[1]}+${w[2]}+${w[03]}`)
```

Reply:
тут зразумела? згодзен?

Reply:
так зразумела

Reply:
згодзен